src/main/java/eu/europeana/metis/sandbox/service/workflow/ExternalValidationService.java

```diff
@@ -1,8 +1,16 @@
 package eu.europeana.metis.sandbox.service.workflow;
 
+import eu.europeana.metis.sandbox.common.exception.RecordProcessingException;
 import eu.europeana.metis.sandbox.domain.Record;
 
 public interface ExternalValidationService {
 
+  /**
+   *
+   * @param record
+   * @return
```

Comment:
Finish the javadoc :)

Reply:
I have it for the next review about validation, but will add it here